scripts/brofish-ping.sh

```diff
@@ -55,11 +58,13 @@ done
 if [[ $ping_ok -ne 1 ]]; then
   cpu=$(brofish_cpu)
 
-  /home/pi/firewalla/scripts/firelog -t cloud -m "brofish ping failed, cpu $cpu, restart brofish now"
+  if [[ $cpu == 'good' ]]; then exit; fi
+
+  /home/pi/firewalla/scripts/firelog -t cloud -m "brofish ping failed, restart brofish now, CPU $cpu"
 
   cd $FIREWALLA_HOME
   $FIREWALLA_HOME/bin/node scripts/diag_log.js \
-    --data '{ "msg": "brofish-ping failed", "broCPU": '"$cpu"' }'
+    --data '{ "msg": "brofish-ping failed", "broCPU": '${cpu%% *}' }'
```

Comment:
If we don't need first part, why do we have to echo it in `brofish_cpu()` line 35 ?

Reply:
for the logging here
https://github.com/firewalla/firewalla/pull/2659/files#diff-d63ae60bfb7eb491d5b288286b10e788R63

Reply:
but the logging part has `${cpu%% *}` in it, which is only the first part (value of CPU inside function), so the echo statement inside the function could be simply `echo $CPU` and then here we just need original line without change(use ${cpu} instead of ${cpu%% *}.